src/timely-util/src/operator.rs

```diff
@@ -207,6 +213,11 @@ where
         E: Data,
         IE: Fn(D1, R) -> (E, R) + 'static,
         R: num_traits::sign::Signed;
+
+    /// Wraps the collection with a passthrough operator that will be shut down when the provided
+    /// token cannot be upgraded anymore. After shutdown all data flowing into the operator will be
+    /// dropped.
```

Comment:
Same remark as above concerning the comment.

Reply:
Fixed!